src/utils/isStandardSyntaxUrl.js

```diff
@@ -1,21 +1,42 @@
-import { hasInterpolation } from "../utils"
+import {
+  hasLessInterpolation,
+  hasPsvInterpolation,
+  hasScssInterpolation,
+} from "../utils"
+
 /**
  * Check whether a URL is standard
  *
  * @param {string} url
  * @return {boolean} If `true`, the url is standard
  */
 export default function (url) {
-  // SCSS or Less interpolation
-  if (hasInterpolation(url)) { return false }
+  if (url.length === 0) { return true }
+
+  // Sass interpolation works anywhere
+  if (hasScssInterpolation(url) || hasPsvInterpolation(url)) { return false }
 
-  // Inside `'` and `"` work only interpolation
+  // Inside `'` and `"` work only LESS interpolation
   if (url[0] === "'" && url[url.length - 1] === "'"
```

Comment:
These subconditions should be wrapped in parens, right?

```js
if ((url[0] === "'" && url[url.length - 1] === "'")
  || (url[0] === "\"" && url[url.length - 1] === "\""))
```

I've faced bugs because of the lack of parens before.